standard_test.go

```diff
@@ -0,0 +1,85 @@
+package retryablehttp
+
+import (
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func TestStandardClient_Do(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != "POST" {
+			t.Fatalf("bad method: %s", r.Method)
+		}
+
+		bytes, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			t.Fatalf("bad body: %s", r.Body)
+		}
+
+		if string(bytes) != "Hello world" {
+			t.Fatalf("bad body: %s", r.Body)
+		}
+
+		w.WriteHeader(200)
+	}))
+	defer server.Close()
+
+	type args struct {
+		req *http.Request
+	}
```

Comment:
This struct seems unnecessary. I'd just have `req` directly in the test case struct.